src/firefly/js/ui/CutoutSizeDialog.jsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+import React, {useEffect} from 'react';
+import {
+    dispatchShowDialog, dispatchHideDialog, getComponentState, dispatchComponentStateChange
+} from '../core/ComponentCntlr.js';
+import CompleteButton from './CompleteButton.jsx';
+import DialogRootContainer from './DialogRootContainer.jsx';
+import {FieldGroup} from './FieldGroup';
+import {PopupPanel} from './PopupPanel.jsx';
+import {useFieldGroupValue, useStoreConnector} from './SimpleComponent';
+import {SizeInputFields} from './SizeInputField';
+
+import HelpIcon from './HelpIcon.jsx';
+import {Stack} from '@mui/joy';
+
+const DIALOG_ID= 'cutoutSizeDialog';
+
+
+export function showCutoutSizeDialog(cutoutParam,dataProductsComponentKey) {
+    const popup = (
+        <PopupPanel title={'Set Cutout Size'}>
+            <FieldGroup groupKey={'cutout-size-dialog'} keepState={true}>
+                <CutoutSizePanelPanel {...{cutoutParam,dataProductsComponentKey}}/>
+            </FieldGroup>
+        </PopupPanel>
+    );
+    DialogRootContainer.defineDialog(DIALOG_ID, popup);
+    dispatchShowDialog(DIALOG_ID);
+}
+
+
+
+
+function CutoutSizePanelPanel({cutoutParam,dataProductsComponentKey}) {
+
```

Comment:
small typo here, should be `CutoutSizePanel`. And then in the call to this component on line 25.